scripts/run_test262_suite.py

```
@@ -518,330 +521,414 @@ def evaluate_suite(
+                    process_returncode = -1
+                    debug_dir = Path(tempfile.gettempdir())
+                    # Stream runner output:
+                    # - stdout inherits this script's fd, so the runner writes
+                    #   directly to our stdout (no Python intermediation).
+                    # - stderr is piped and drained on a background thread so we
+                    #   can both forward it live to this script's stderr AND
+                    #   capture it for fallback diagnostic display when the
+                    #   runner exits without a parseable JSON report.
+                    captured_stderr_chunks: list[str] = []
+                    try:
+                        process = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)